js/woo.js

```diff
@@ -1375,6 +1380,44 @@ module.exports = class woo extends Exchange {
         return this.safeString (statuses, status, status);
     }
 
+    async transfer (code, amount, fromAccount, toAccount, params = {}) {
+        await this.loadMarkets ();
+        const currency = this.currency (code);
+        const request = {
+            'token': currency['id'],
+            'amount': this.parseNumber (amount),
+            'from_application_id': fromAccount,
+            'to_application_id': toAccount,
+        };
+        const response = await this.v1PrivatePostAssetMainSubTransfer (this.extend (request, params));
+        //
+        //     {
+        //         "success": true,
+        //         "id": 200
+        //     }
+        //
+        const transferOptions = this.safeValue (this.options, 'transfer', {});
+        const fillResponseFromRequest = this.safeValue (transferOptions, 'fillResponseFromRequest', true);
+        const transfer = {
+            'info': response,
+            'id': this.safeString (response, 'id'),
+            'timestamp': undefined,
+            'datetime': undefined,
+            'currency': undefined,
+            'amount': undefined,
+            'fromAccount': undefined,
+            'toAccount': undefined,
+            'status': this.safeString (response, 'success') ? 'ok' : 'failed',
```

Comment:
Normally this would go inside `parseTransferStatus` however `parseTransferStatus` is already defined for a different response.

Reply:
We can still reuse `parseTransferStatus` for this purpose, what are the possible values and types for `success`? Is it a string or a boolean? It would be great if you include samples of a complete response in `transfer` and of a single parsed transfer in `parseTransfer`.